packages/usa-file-input/src/index.js

```diff
@@ -12,7 +12,6 @@ const BOX_CLASS = `${PREFIX}-file-input__box`;
 const INSTRUCTIONS_CLASS = `${PREFIX}-file-input__instructions`;
 const PREVIEW_CLASS = `${PREFIX}-file-input__preview`;
 const PREVIEW_HEADING_CLASS = `${PREFIX}-file-input__preview-heading`;
-const DISABLED_CLASS = `${PREFIX}-file-input--disabled`;
```

Comment:
Right now, adding `aria-disabled="true"` to the input element does not cause the component to look disabled. Since this class is no longer available, we should make sure that `aria-disabled` triggers the same styling as `disabled`.

Reply:
I've brought over some of the javascript and storybook settings from #5063 so that the component will dynamically add the `aria-disabled` attribute to the parent class in the same way it currently does with `disabled`

Now when the input has the attribute, the `usa-file-input` div will receive it as well and style it appropriately.

If you manually add `aria-disabled="true"` to the input after init it will not inherit the same styles, but this is to be expected because of the way the component is built and styled through the new divs and is consistent with how the component was styled previously